### DISCUSSION TITLE
👁️ I want to hide packages as logged user

### DISCUSSION THREAD

**Discussion by landsman** [npm/feedback#612] [Website] [answered]
Hi guys,

my scenario:

1. I am looking for package to resolve my problem X
2. There are several packages with very similar name
3. I tried a few packages and I want to "flag them" somehow that I already tried them

So basically I am looking for "hide" functionality for logged user.
What do you think?

From my perspective it's next value for npm users, you can not do this on Github, so.

Thanks!

**Comment by MylesBorins** [comment, marked as answer]
hey @landsman this is definitely an interesting idea but not something that we are going to pursue at this time. There would be quite a bit of state management and a need for UX to review and revert mistakes in case you accidentally ignored something you didn't mean to. At this time we are hyper focused on securing the npm supply chain and improving account security and as such we would not prioritize this feature request.

**Reply by landsman** [reply]
ok, understood, thanks for the reply.
